x-pack/test/functional/apps/security/rbac_phase1.js

```diff
@@ -0,0 +1,135 @@
+/*
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+* or more contributor license agreements. Licensed under the Elastic License;
+* you may not use this file except in compliance with the Elastic License.
+*/
+
+import expect from 'expect.js';
+import { indexBy } from 'lodash';
+export default function ({ getService, getPageObjects }) {
+
+  const PageObjects = getPageObjects(['security', 'settings', 'monitoring', 'discover', 'common', 'reporting', 'visualize', 'header']);
+  const log = getService('log');
+  const esArchiver = getService('esArchiver');
+  const remote = getService('remote');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('rbac ', async function () {
+    before(async () => {
+      await remote.setWindowSize(1600, 1000);
+      log.debug('users');
+      await esArchiver.loadIfNeeded('logstash_functional');
+      log.debug('load kibana index with default index pattern');
+      await esArchiver.load('discover');
+      await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'UTC', 'defaultIndex': 'logstash-*' });
+      await PageObjects.settings.navigateTo();
+      await PageObjects.security.clickElasticsearchRoles();
+      await PageObjects.security.addRole('rbac_all', {
+        "applications": [
+          {
+            "application": "kibana",
+            "resources": ["default"],
```

Comment:
nit: we technically aren't using the "default" resource anymore, and we're using "*", but the application and resources parameters are unused currently. I'd probably change this syntax to just:
```
await PageObjects.security.addRole('rbac_all', {
        "kibana": ["all"],
       "indices": [{
          "names": [ "logstash-*" ],
          "privileges": [ "read" ]
        }]
      });
```

Reply:
done. Also added the `view_index_metadata` to the logstash index which shouldn't matter for this test.